src/invidious/routes/api/v1/channels.cr

```diff
@@ -262,6 +262,64 @@ module Invidious::Routes::API::V1::Channels
     end
   end
 
+  def self.podcasts(env)
+    locale = env.get("preferences").as(Preferences).locale
+
+    env.response.content_type = "application/json"
+
+    ucid = env.params.url["ucid"]
+    continuation = env.params.query["continuation"]?
+
+    # Use the macro defined above
+    channel = nil # Make the compiler happy
+    get_channel()
+
+    items, continuation = fetch_channel_podcasts(channel.ucid, channel.author, continuation)
+
+    JSON.build do |json|
+      json.object do
+        json.field "playlists" do
+          json.array do
+            items.each do |item|
+              item.to_json(locale, json) if item.is_a?(SearchPlaylist)
+            end
+          end
+        end
+
+        json.field "continuation", continuation
```

Comment:
Can you please separate `continuation` (user-provided data) from `next_continuation` (new ctoken provided by YT).